### PR TITLE
Google Season of Docs organization applications

### PR DESCRIPTION
**Pull request by JyeSmith**
https://developers.google.com/season-of-docs

### CODE REVIEW

**Review by FOG-Yamato — APPROVED** [review]

**Review by StonedDawg — CHANGES_REQUESTED** [review]
put it as a list here ( - ) to match the above list.

**Review by StonedDawg — COMMENTED** [review]
put it as a list (- ) here to match the above list

File: docs/gsod.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- Introduce the Technical Writer to our troubleshooting procedures.
+- This experience will help to identify any knowledge gaps in existing user ELRS documentation.
+- Devs and Advanced Users will provide information on general user queries from project volunteers and assist with documentation on recurring issues.
+- Review documentation used to assist manufacturers where English may not be their first language.
+- Hopefully assist in a major release (ELRS V3) and release notes.
+
+We estimate that this work will take 4-6 months to complete.
+
+Work that is out-of-scope of ELRS project:
+In-depth PID Tuning of RC vehicles (Multicopter, Fixed - Wing, Boat, Rover).
```
Comment (StonedDawg) [inline docs/gsod.md]: put a list prefix ( - ) here to match the above list

**Review by StonedDawg — APPROVED** [review]

**Comment by tapaswenipathak** [comment]
hi folks, how should someone contact you, if they are interested?

**Pull request merged**